wfe2/wfe.go

```diff
@@ -941,14 +936,21 @@ func (wfe *WebFrontEndImpl) postChallenge(
 		return
 	}
 
-	var challengeUpdate core.Challenge
+	// NOTE(@cpu): Historically a challenge update needed to include
+	// a KeyAuthorization field. This is no longer the case, since both sides can
+	// calculate the key authorization as needed. We unmarshal here only to check
+	// that the POST body is valid JSON. Any data/fields included are ignored to
+	// be kind to ACMEv2 implementations that still send a key authorization.
+	var challengeUpdate interface{}
```

Comment:
I think if we make this an empty struct it would achieve the goal, and maybe be a little clearer?

Reply:
Sure WFM!

Reply:
Resolved in 3e1e460